src/components/legend/draw.js

```diff
@@ -277,26 +277,27 @@ module.exports = function draw(gd) {
         // legend, background and border, scroll box and scroll bar
         legend.attr('transform', 'translate(' + lx + ',' + ly + ')');
 
-        // increase the background and clip-path width
-        // by the scrollbar width and margin
+        // This attribute is only needed to help the jasmine tests
+        legend.attr('data-height', legendHeight);
```

Comment:
Let's not pollute our `<svg>` for the sake of our jasmine tests.

There's got to be a better way to test this.